src/WebJobs.Script/Utility.cs

```diff
@@ -792,6 +793,28 @@ public static bool IsMediaTypeOctetOrMultipart(MediaTypeHeaderValue mediaType)
             }
         }
 
+        public static void LogAutorestGeneratedJson(string rootScriptPath, ILogger logger = null)
+        {
+            string autorestGeneratedJsonPath = Path.Combine(rootScriptPath, ScriptConstants.AutorestGeenratedMetadataFileName);
+            JObject autorestGeneratedJson;
+
+            if (FileUtility.FileExists(autorestGeneratedJsonPath))
+            {
+                string autorestGeneratedJsonPathContents = FileUtility.ReadAllText(autorestGeneratedJsonPath);
```

Comment:
for adding unit tests, create a file in temp directory: https://stackoverflow.com/questions/278439/creating-a-temporary-directory-in-windows

and you can use `TestLogger` - you find references in other tests.

Reply:
Done. Thanks.